src/extension/background-script/actions/lnurl/authOrPrompt.ts

```diff
@@ -33,10 +33,17 @@ async function authOrPrompt(
   // we have the check the unlock status manually. The account can still be locked
   // If it is locked we must show a prompt to unlock
   const isUnlocked = await state.getState().isUnlocked();
+  const account = await state.getState().getAccount();
 
   // check if there is a publisher and lnurlAuth is enabled,
   // otherwise we we prompt the user
-  if (isUnlocked && allowance && allowance.enabled && allowance.lnurlAuth) {
+  if (
+    isUnlocked &&
+    allowance &&
+    allowance.enabled &&
+    allowance.lnurlAuth &&
+    account?.mnemonic
```

Comment:
Unfortunately this will break legacy LNURL-auth. A mnemonic is only mandatory for the Alby connector. For the other connectors, they can work with old signing functions (although it would be nice for them to use the master key instead)

Reply:
added support for legacy lnurl-auth support, they can now login without setting up keys